indico/web/client/js/jquery/widgets/jinja/ckeditor_widget.js

```diff
@@ -5,37 +5,24 @@
 // modify it under the terms of the MIT License; see the
 // LICENSE file for more details.
 
-/* global RichTextEditor:false, $E:false */
+import ClassicEditor from 'ckeditor';
```

Comment:
Add this to your vscode workspace settings so you get warnings even for files that are in one of the blacklisted folders:

```json
		"eslint.options": {
			"ignore": false,
		},
```